exe/wallet/Main.hs

```diff
@@ -122,6 +135,7 @@ Usage:
   cardano-wallet wallet get [--port=INT] <wallet-id>
   cardano-wallet wallet update [--port=INT] <wallet-id> --name=STRING
   cardano-wallet wallet delete [--port=INT] <wallet-id>
+  cardano-wallet transaction create [--port=INT] --wallet-id=STRING --payment=STRING...
```

Comment:
I wonder if we could put a `--payment=PAYMENT` here (to convey the idea that this has a particular format), or maybe, add an `examples` section to the docopt definition to give some example usage :thinking: ?